docs/cli/kops_replace.md

```diff
@@ -5,7 +5,7 @@ Replace cluster resources.
 ### Synopsis
 
 
-Replace a resource specification by filename or stdin.
+Replace a resource configuration by filename or stdin.
```

Comment:
Looking at what kubectl does, it just says `Replace a resource by filename or stdin. `

That does avoid the issue :-)